frontend/public/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -37,7 +37,7 @@ export const OperatorHubItemDetails: React.SFC<OperatorHubItemDetailsProps> = ({
 
   const getHintBlock = () => {
     if (installed) {
-      const filterName = _.get(item.obj, 'status.packageName', item.obj.metadata.name);
+      const filterName = _.get(item.obj, 'status.channels[0].currentCSV', '');
```

Comment:
Looks like @alecmerdler  changed it from `channels[0].currentCSV` to `packageName` in #1327

Reply:
We could simply take the name filter out if it's not reliable.